catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -259,7 +259,7 @@
                 value="The workspace '%[attribute=title]' (id: %[attribute=id]) contains up to %[hitCount] query hits."/>
         <argument value="%[attribute=title] (id: %[attribute=id]): workspace query notice"/>
         <argument value="donotreply@test.com"/>
-        <argument value="connexta-com.mail.protection.outlook.com"/>
+        <argument value="example.mail.protection.outlook.com"/>
```

Comment:
Is there a reasonable default value for the mail host? I'm thinking it should be an empty string.